### DISCUSSION TITLE
Réunion en ligne de préparation du congrès de la SMQ

### DISCUSSION THREAD

**Discussion by franckcalard** [ouvroir/museadon#70] [General]
En prévision de notre future communication au Congrès de la SMQ, il faudrait que l'on puisse se rencontrer afin de rédiger le texte qui sera présenté au congrès.

La présence des étudiants du séminaire de Monsieur Château-Dutier n'est pas obligatoire, sentez-vous libre de venir - ou pas - si vous êtes disponibles !

Dans le meilleur des cas, il faudrait deux rencontres de 1h30 par Zoom. Vous pouvez aussi être disponible que pour l'une des deux rencontres.

Afin de connaître vos disponibilités, voici un lien framadate :

https://framadate.org/hdoPLE4SOX4u5Djg

 N'hésitez pas à modifier les créneaux indiqués voire à en ajouter et/ou à répondre à cette discussion si vous avez des questions.

Belle journée/soirée :) 
@ouvroir/museadon

**Comment by emchateau** [comment]
Bonjour, je m’aperçois que nous n’avons pas fait de suivi pour la réunion. Je vois qu’il reste encore ce soir où plusieurs d’entre vous étiez libre à partir 19h30
Est-ce que nous pourrions nous rencontrer ?
@ouvroir/museadon

**Reply by Alissia-CptBath** [reply]
Je suis disponible , j'attends les réponses de l'ensemble des participants. @ouvroir/museadon

**Reply by franckcalard** [reply]
Je suis aussi disponible ce soir ! @ouvroir/museadon

**Reply by JuliettePassilly36** [reply]
Je suis disponible également ! Ce soir, je suis disponible de 16h45 à 19h  ou à partir de 19h45 !

**Reply by JulienLehoux** [reply]
Bonjour, je me suis personnellement tenu loin des discussions mais histoire d'enlever toute ambiguïté dans mon cas, je ne pourrais pas participer à ce congrès. Je suis plus qu'occupé et je ne peux m'engager pour cette présentation. Bonne chance cependant pour ce congrès et je reste ouvert pour toute implication avec ce projet dans le futur !

**Reply by Alissia-CptBath** [reply]
La réunion est elle maintenue ce soir ?

**Reply by emchateau** [reply]
Oui je pense qu’il faut essayer. Ceux d’entre-vous qui le souhaite nous pourrions nous retrouver à 19h30 pour une première discussion Emmanuel Chateau Dutier vous invite à une réunion Zoom planifiée.
https://umontreal.zoom.us/j/99237515427?pwd=Uk1Qd3NoK2ljQkJ6QXkrTldhU05jdz09

**Comment by Alissia-CptBath** [comment]
Bonjour/Bonsoir à tous,

Nous avons fait une première réunion pour le colloque de la SMQ.  Nous avons commencé à rédiger un petit texte qu’il faudrait compléter pour la communication du 14 octobre.

Vous êtes tous invités – ce n’est pas une obligation - à compléter le document via le lien suivant : (https://demo.hedgedoc.org/R-3ZG8IwQ3u8jcmTkCkt4g?edit) . Afin de compléter le document, vous devrez vous authentifier avec GitHub. Cela permettait de recueillir les idées de l’ensemble du groupe, même pour ceux qui ne pourraient pas participer à la présentation, mais qui souhaitaient apporter une contribution. 

Nous allons aussi travailler avec Canvas pour faire la diapositive, si l’un de vous souhaite s’en charger exclusivement ou participer, merci de l’indiquer.

Enfin, nous allons faire une deuxième réunion le jeudi 30 septembre à partir de 14h. N’hésitez pas à participer, ou nous rejoindre en cours de réunion, vous êtes tous les bienvenus.

À très bientôt, belle soirée à vous.
Alissia
@ouvroir/museadon

**Reply by Alissia-CptBath** [reply]
Précisions : à côté de "Hedge doc " en haut à gauche, il y a 3 icônes : une pour voir , une pour voir et modifier et une pour modifier le document.

**Reply by Alissia-CptBath** [reply]
@emchateau @ouvroir/museadon Doit-on repasser par le même liens que la dernière fois pour avoirs accès au zoom à 14h ? [](https://umontreal.zoom.us/j/99237515427?pwd=Uk1Qd3NoK2ljQkJ6QXkrTldhU05jdz09 )

**Reply by emchateau** [reply]
Désolé je suis allé vite, car j'étais au téléphone en même temps que j’écrivais. Oui c’est bon, salutations !

**Comment by Alissia-CptBath** [comment]
@ouvroir/museadon Seriez vous disponible pour faire une réunion dimanche 3 octobre le soir à 19h30?

**Comment by emchateau** [comment]
Salut,
Pour moi c’est possible dimanche à 19h30.
Bien cordialement,
Emmanuel

**Comment by Alissia-CptBath** [comment]
Bonjour,

J'ai crée un deuxième sondage qui couvre les dates à partir de la semaine du lundi 4 octobre pour nos réunions si jamais nous ne pouvons pas terminer avant notre rédaction de texte + diapo pour le colloque de la SMQ.
Merci d'ajouter vos nouvelles disponibilités ici : [https://framadate.org/tuqptG45ZDSrpbMf](https://framadate.org/tuqptG45ZDSrpbMf ).
Petit rappel : vous pouvez toujours ajouter vos suggestions dans le HedgeDoc même si vous ne souhaitez pas participer aux réunions, n'hésitez pas. Egalement, vous êtes évidement les bienvenus pour les réunions zoom.

J'ai vue que @JuliettePassilly36 et @franckcalard serait éventuellement disponible dimanche 3 octobre à 19h30, est ce toujours le cas ?
Emmanuel, avez vous crée un liens Figma pour la dispositive ? @emchateau 

Merci,

@ouvroir/museadon

**Comment by emchateau** [comment]
Salut,
Je suis libre ce soir et j’ai renseigné mes disponibilités la semaine prochaine.
Je n’ai pas encore eu le temps de passer les visuels sur Figma.
Bien cordialement,

**Reply by Alissia-CptBath** [reply]
@ouvroir/museadon Juliette n'est pas disponible aujourd'hui, donc je pense qu'il faut repousser ?

**Comment by emchateau** [comment]
Pas de pb pour faire ça en début de semaine.
Bien cordialement,
Emmanuel

**Reply by franckcalard** [reply]
Bonsoir, je n'étais pas disponible également, je vais tenter de mettre mes disponibilités sur le framadate dans la soirée merci !
@ouvroir/museadon

**Reply by Alissia-CptBath** [reply]
Bonjour,
d'après le framadate nous pourrions nous réunir soit ce soir à 19h30 ou demain matin à 8h , que préférez-vous ?
@ouvroir/museadon

**Reply by franckcalard** [reply]
Je préfère demain matin, j'aime être debout tôt et avoir ma journée en perspective !
Je reste à votre écoute ;) 
@ouvroir/museadon

**Reply by Alissia-CptBath** [reply]
Je préfère demain également :)

**Comment by emchateau** [comment]
Entendu pour demain matin.
Je serai seulement avec vous de 8h à 8h30 car j’ai accepté entre temps un rdv. Mais je pourrai vous retrouver un peut après je pense vers 9h.
Bien cordialement,
Emmanuel

**Reply by Alissia-CptBath** [reply]
@emchateau comme c'est votre liens zoom ; envoyez nous une notification quand vous serez disponible peut-être ?

**Comment by emchateau** [comment]
Tout est beau. Le lien reste ouvert. Et je serai là à 8h.
Bien cordialement,
Emmanuel

**Comment by emchateau** [comment]
Emmanuel Chateau Dutier vous invite à une réunion Zoom planifiée.

Participer à la réunion Zoom
https://umontreal.zoom.us/j/99237515427?pwd=Uk1Qd3NoK2ljQkJ6QXkrTldhU05jdz09

**Reply by franckcalard** [reply]
Bonjour, j'ai essayé de me connecter mais impossible car je n'ai pas le code secret.... À cela s'ajoute mes perpétuels problèmes de connexion... Malheureusement, je n'ai pas réussi à vous rejoindre ! Pouvez-vous me dire ce que vous avez fait aujourd'hui ? Mercie et encore désolé :/

**Reply by Alissia-CptBath** [reply]
Bonjour Franck, 
Nous avons retravaillé le texte dans le HedgeDoc, pas terminée mais en bonne voie. + Commencé à regarder pour Figma et créer la diapositive (tu as eu une invitation dans tes courriels pour accéder à figma).

**Comment by emchateau** [comment]
Bonjour,
Normalement Alissia et Juliette sont encore en ligne 

https://umontreal.zoom.us/j/7563708702?pwd=UFlsQm5JM1pKN3pwcnBETzdFRFdyUT09

ID de réunion : 756 370 8702
Code secret : archi

**Comment by Alissia-CptBath** [comment]
Prochaines dates pour une réunion d'après le framadate :
mercredi 6 octobre à  partir de 19h30
vendredi 8 octobre à partir de 17h ou à partir de 19h30.

Personnellement, vendredi à partir de 17h m'arrange mieux.

@franckcalard @emchateau

**Reply by franckcalard** [reply]
Je suis disponible aux deux dates, je préfère également le vendredi à partir de 17h !

**Comment by emchateau** [comment]
Bonsoir,
Demain soir je ne pourrai finalement pas car j’ai des amis à souper.
Cela ne vous empêche pas de travailler ensemble.
Je pourrais regarder ensuite.
Bien cordialement,
Emmanuel

**Reply by Alissia-CptBath** [reply]
@franckcalard @emchateau êtes vous disponible demain soir à partir de 17h ou 18h pour une réunion ?  bonne soirée

**Reply by franckcalard** [reply]
Je suis disponible !

**Comment by emchateau** [comment]
Salut,
C’est bon pour moi demain soir.
Bien cordialement,
Emmanuel

**Reply by franckcalard** [reply]
Bonjour, je serai disponible à partir de 18h ce soir, je ne sais pas si ca arrange Alissia comme tu avais un rendez-vous ! À ce soir :) @Alissia-CptBath @emchateau

**Reply by Alissia-CptBath** [reply]
Bonjour, c'est correct pour moi 18h, pas de problèmes @franckcalard @emchateau

**Comment by emchateau** [comment]
Je serai là 

Emmanuel Château

**Comment by emchateau** [comment]
Même lien https://umontreal.zoom.us/j/99237515427?pwd=Uk1Qd3NoK2ljQkJ6QXkrTldhU05jdz09

**Reply by emchateau** [reply]
Salut à toutes et tous, voici le document pour les diapositives. Je me suis un peu amusé à la fin, nous ne sommes pas obligé. Peut-être préfèrerez-vous la citation fort à propos choisie par Alissia dans le texte. https://demo.hedgedoc.org/s/37-jJdhQU#
[museadon2021.pdf](https://github.com/ouvroir/museadon/files/7314735/museadon2021.pdf)

**Reply by Alissia-CptBath** [reply]
Je préfère la citation que j'ai mise dans le texte mais je ne sais pas si c'est finalement approprié comme vous me disiez que le fondateur de Wordpress n'était pas vraiment en accord avec le genre de projet que nous défendons dans la communication. @emchateau

**Reply by Alissia-CptBath** [reply]
J'ai fais un dernier framapad avant le colloque de jeudi [https://framadate.org/71KrG7uctMPcMSFZ](https://framadate.org/71KrG7uctMPcMSFZ) pour que nous puissions finir les derniers détails et répéter le texte. @franckcalard @emchateau 
Bonne soirée, bon week end,

**Comment by JuliettePassilly36** [comment]
Bonjour ! 

J'espère que vous allez tous bien ! Comment s'organise-t-on pour demain ? Que faut-il prévoir pour les entrées à la conférence ? Est ce qu'on y va tous ensemble ou se rejoint-on sur place ? 
 Et est-ce qu'il y a encore besoin de travail sur le texte/diapo ? 

Désolée pour cette missive de questions ^^'

Je reste à disposition, 

A demain :)

**Reply by emchateau** [reply]
Bonjour Juliette,
Ainsi que je l’avais expliqué lors de notre première rencontre, malheureusement la SMQ a refusé de proposer des places gratuites au congrès. Seuls les intervenants auront une gratuité pour accéder à leur session. De mon côté je paye ma place. En fait, le nombre de places était limité en raison des conditions sanitaires, mais j’ai essayé de négocier des places en lignes, mais en vain. J’en suis vraiment très très déçu.
J’aurais dû demander pour la salle de réseautage à 17h. Mais là, vous devriez pouvoir y accéder. Je vais essayer de me le faire confirmer. 
Désolé à nouveau de ces mauvaises nouvelles. Il y a décidément un travail à faire avec la SMQ pour que les étudiantes et les étudiants y aient plus de place.
Emmanuel
